Package.swift

```diff
@@ -28,6 +28,7 @@ let package = Package(
   products: [
     .library(
       name: "Testing",
+      type: .dynamic,
```

Comment:
The workaround in #816 was just the compilation guard in the test file below. Is changing this to `.dynamic` actually necessary?

Reply:
Yep, otherwise the symbols aren't exported. @dschaefer2 has more info.

Reply:
Can you please add a comment capturing why this is needed? I anticipate it may be forgotten since the current motivation for adding it is a platform-specific workaround

Reply:
Added.